composer.json

```diff
@@ -127,14 +127,23 @@
             "drush config:set --yes system.logging error_level verbose",
             "drush pm:enable --yes default_content"
         ],
+        "drupal:quick-start": [
+            "Composer\\Config::disableProcessTimeout",
+            "@php web/core/scripts/drupal quick-start"
+        ],
         "post-create-project-cmd": [
             "@drupal:install",
-            "Composer\\Config::disableProcessTimeout",
-            "test -n \"$CI\" || php -d max_execution_time=0 web/core/scripts/drupal quick-start"
+            "test -n \"$CI\" || @drupal:quick-start"
         ]
     },
     "scripts-descriptions": {
         "drupal:install": "Installs Drupal CMS.",
-        "drupal:install-dev": "Installs Drupal CMS, with additional modules and configuration tweaks for development."
+        "drupal:install-dev": "Installs Drupal CMS, with additional modules and configuration tweaks for development.",
+        "drupal:quick-start": "Installs Drupal CMS (if not already installed) and runs the PHP webserver."
+    },
+    "scripts-aliases": {
+        "drupal:quick-start": [
+            "qs"
+        ]
```

Comment:
If we're gonna do this, how about we also add an alias for `drupal:install`, which is a user-facing command?

Reply:
Sure. I'll just do what Drush does: `si` and `sin`.